### DISCUSSION TITLE
Did .GiftWrapPrice go away?

### DISCUSSION THREAD

**Discussion by wheelersf** [abuzuhri/Amazon-SP-API-CSharp#53] [General]
Hi,

Old MWS used to have an item node of  'orderitem.GiftWrapPrice.Amount'.  Did this go away? I'm unable to find a reference at the order or item-level object.

Thanks,
Scott

**Comment by abuzuhri** [comment]
Hi @wheelersf ,

I can see its still exist buy under BuyerInfo

https://github.com/amzn/selling-partner-api-docs/blob/main/references/orders-api/ordersV0.md#orderitembuyerinfo

**Comment by wheelersf** [comment]
Hi Tareq, 

I can see the gift information in the ItemBuyerInfo object, but am getting the following exception message when attempting to iterate each item in the list:

Error	CS1579	foreach statement cannot operate on variables of type 'OrderItemsBuyerInfoList' because 'OrderItemsBuyerInfoList' does not contain a public instance or extension definition for 'GetEnumerator'

I don't get this exception when iterating orders, items, etc. using the same coding convention.

Thanks,
Scott

```
//Get Items Buyer Info
var orderItemsBuyerInfoList = amazonConnection.Orders.GetOrderItemsBuyerInfo(order.AmazonOrderId);
                       
foreach (var ItemInfo in orderItemsBuyerInfoList)
{
    //do work...
}
```

**Comment by abuzuhri** [comment]
Hi @wheelersf 

Please upgrade NuGet to v1.0.29 , and let me if that solve the problem

**Comment by wheelersf** [comment]
Hi Tareq,

I'm actually getting the same exception message after updating to v1.0.29.

Thanks,
Scott

**Reply by abuzuhri** [reply]
> //Get Items Buyer Info
> var orderItemsBuyerInfoList = amazonConnection.Orders.GetOrderItemsBuyerInfo(order.AmazonOrderId);
>                        
> foreach (var ItemInfo in orderItemsBuyerInfoList)
> {
>     //do work...
> }


Please change your code to 

            foreach (var ItemInfo in orderItemsBuyerInfoList.OrderItems)
            {
                //do work...
            }

**Comment by abuzuhri** [comment]
```CSharp
        foreach (var ItemInfo in orderItemsBuyerInfoList.OrderItems)
        {
            //do work...
        }
```

**Comment by wheelersf** [comment]
@abuzuhri Thank you, Sir!  That did it.
